mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -97,6 +97,26 @@ class RFormula(override val uid: String)
   /** @group setParam */
   def setLabelCol(value: String): this.type = set(labelCol, value)
 
+  /**
+   * Force to index label whether it is numeric or string type.
+   * Usually we index label only when it is string type.
+   * If the formula was used by classification algorithms,
+   * we can force to index label even it is numeric type by setting this param with true.
+   * Default: false.
+   */
+  @Since("2.0.0")
+  val indexLabel: BooleanParam = new BooleanParam(this, "indexLabel",
```

Comment:
Rename to "forceIndexLabel" since we can still index String labels when indexLabel=false